sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala

```diff
@@ -408,7 +408,15 @@ private[spark] class HiveExternalCatalog(conf: SparkConf, hadoopConf: Configurat
   }
 
   private def defaultTablePath(tableIdent: TableIdentifier): String = {
-    val dbLocation = getDatabase(tableIdent.database.get).locationUri
```

Comment:
I'd like the first one, `defaultTablePath` in `SessionCatalog` has been used in `CreateDataSourceAsSelectCommand`, I think it is useful to keep it in `SessionCatalog`, maybe later in another place we want to use it.